.changeset/five-jeans-perform.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools/sync-actions': patch
+---
+
+Add support for StandalonePrice `setPriceTiers`
```

Comment:
this is not what this patch is doing. 
`rollback setPriceTiers name change` or something along those lines better explains what is going on